react/src/components/ImageTags.tsx

```diff
@@ -85,48 +84,49 @@ interface LangTagsProps extends TagProps {
 }
 export const LangTags: React.FC<LangTagsProps> = ({ image, ...props }) => {
   image = image || '';
-  const [, { getImageLang, tagAlias }] = useBackendAIImageMetaData();
+  const [, { getImageLang }] = useBackendAIImageMetaData();
   return (
     <Tag color="green" {...props}>
-      {tagAlias(getImageLang(image))}
+      {getImageLang(image)}
```

Comment:
The reason for removing tagalias is that sometimes it is necessary to show the language without using the LangTags component, so I think it is better to include tagAlias inside the getImageLang function.